lint.js

```diff
@@ -80,10 +85,14 @@ More information: https://speccy.io/rules/#${rule.name}
 
 const command = async (file, cmd) => {
     const verbose = cmd.quiet ? 1 : cmd.verbose;
+    let rules = [].concat(cmd.rules);
```

Comment:
No need to concat a single array with itself. This is just making a copy as there is only one array there.

Reply:
fixed